modules/Timetable Admin/courseEnrolment_manage_byPerson.php

```diff
@@ -138,7 +138,7 @@
             });
         $table->addColumn('roleCategory', __('Role Category'))
             ->format(function($person) {
-                return __($person['roleCategory']) . '<br/><small><i>'.Format::userStatusInfo($person).'</i></small>';
+                return __($person['roleCategory']) . '<br/><small><i>'.__(Format::userStatusInfo($person)).'</i></small>';
```

Comment:
It looks like Format::userStatusInfo handles much of the translation inside the method, in this case it may work best to add the `__()` to the `$person['staffType'];` output in Format.php line 461.

Reply:
Hi Sandra, it's correct! I made the changes.